docs/cody/capabilities/chat.mdx

```diff
@@ -37,9 +37,9 @@ For example, ask Cody "What does this file do?"
 
 Cody will take a few seconds to process your question, providing contextual information about the files it reads and generating the answer.
 
-        <video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto' }}>
-                    <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/ask-cody-question.mp4" type="video/mp4" />
-                    </video>
+<video width="1920" height="1080" loop playsInline controls style={{ width: '100%', height: 'auto', aspectRatio: '1920 / 1080' }}>
+  <source src="https://storage.googleapis.com/sourcegraph-assets/Docs/Media/chat-interface-0724.mp4" type="video/mp4" />
```

Comment:
can we make a note that this is only available for PLG for now? Enterprise will come on August 7. 

and then follow up to remove this disclaimer after 😇